lib/experiment/local/client.rb

```diff
@@ -22,6 +25,16 @@ def initialize(api_key, config = nil)
                       end
       @fetcher = LocalEvaluationFetcher.new(api_key, @logger, @config.server_url)
       raise ArgumentError, 'Experiment API key is empty' if @api_key.nil? || @api_key.empty?
+
+      @assignment_service = nil
+      # rubocop:disable Style/GuardClause
+      if config&.assignment_config
+        # rubocop:enable Style/GuardClause
```

Comment:
Do we still need these with the rubocop config changes?

Reply:
Fixed styling :)